src/crystal/system/unix/file_descriptor.cr

```diff
@@ -80,7 +80,7 @@ module Crystal::System::FileDescriptor
   end
 
   private def system_pos
-    pos = LibC.lseek(fd, 0, IO::Seek::Current)
+    pos = LibC.lseek(fd, 0, IO::Seek::Current).to_i64!
```

Comment:
Why does it need <code>to_i64<b>!</b></code> if `to_i64` already introduces no additional runtime checks for types <= 64

Reply:
I accociate the exclamation mark method when I know the cast is safe and doesn't need any check. But we can use `#to_i64` as well. I'm not sure I care either way.

Reply:
We happen to live in a world where we can say that this will never be `Int128`. But why rely on that instead of just writing the expression that's always correct? I think those `!` methods need a high bar for acceptance.

Reply:
Ha, on the other hand maybe even if it was `Int128` we'd prefer to assume that there are never such huge files. Whatever :D